content/docs/started/workstation/minikube-linux.md

```diff
@@ -0,0 +1,304 @@
+ml-pipeline-79b4f85cbc-8tjff                                   1/1     Running     0        9m55s
+ml-pipeline-ml-pipeline-visualizationserver-5fdffdc5bf-zsngc   1/1     Running     0        9m42s
+ml-pipeline-persistenceagent-645cb66874-b465h                  1/1     Running     0        9m54s
+ml-pipeline-scheduledworkflow-6c978b6b85-cffng                 1/1     Running     0        9m42s
+ml-pipeline-ui-6995b7bccf-s642q                                1/1     Running     0        9m45s
+ml-pipeline-viewer-controller-deployment-8554dc7b9f-vgw9r      1/1     Running     0        9m44s
+mysql-598bc897dc-lsmqc                                         1/1     Running     0        9m54s
+notebook-controller-deployment-7db57b9ccf-vw9vw                1/1     Running     0          10m
+profiles-deployment-5d87dd4f87-7gfrj                           2/2     Running     0        9m41s
+pytorch-operator-5fd5f94bdd-kfgvp                              1/1     Running     2          10m
+seldon-controller-manager-679fc777cd-4n229                     1/1     Running     0        9m39s
+spark-operatorcrd-cleanup-kxr7g                                0/2     Completed   0          10m
+spark-operatorsparkoperator-c7b64b87f-cfptj                    1/1     Running     0          10m
+spartakus-volunteer-6b767c8d6-4v6hc                            1/1     Running     0          10m
+tensorboard-6544748d94-8ctk8                                   1/1     Running     0          10m
+tf-job-operator-7d7c8fb8bb-xz5pw                               1/1     Running     1          10m
+workflow-controller-945c84565-57c72                            1/1     Running     0          10m
+``` 
+
+## Launch of Kubeflow central dashboard
+Kubeflow Dashboard can be accessed via istio-ingressgateway service. To see your setting of istio-ingressgateway service, execute the following:
+```
+export INGRESS_HOST=$(minikube ip)
+export INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="http2")].nodePort}')
+```
+Then you can access Kubeflow dashboard in a web browser: 
+```
+http://<INGRESS_HOST>:<INGRESS_PORT>
+```
+
+## Execution of a MNIST on-prem notebook
+
+[The MNIST on-prem notebook](https://github.com/kubeflow/fairing/blob/master/examples/mnist/mnist_e2e_on_prem.ipynb) builds a docker image, launches a TFJob to train model, and creates a InferenceService (KFServing) to deploy the trained model.
+
+### Prerequistes
+
+#### Step 1: Set up Python environment
+Python 3.5 or later is required. 
+```
+apt-get update; apt-get install -y wget bzip2
+wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+bash Miniconda3-latest-Linux-x86_64.sh 
+```
+Re-open your current shell.
+Create a Python 3.7 environment name of `mlpipeline` (or any name you preferred):
+```
+conda create --name mlpipeline python=3.7
+conda init
+conda activate mlpipeline 
+``` 
+#### Step 2: Install Jupyter Notebook
```

Comment:
I'm confused here. Please correct me if I've got this wrong. (I've never used Minikube.) My expectation is that the user would run this notebook from within their Kubeflow cluster, using the Kubeflow **Notebook Servers** option in the UI. For that reason, I'm wondering why I need to install Jupyter at this point. (If the Jupyter installation isn't necessary, there may also be other things we don't need to tell them to install, such as Python?)

Reply:
@sarahmaddox 

I checked with the owner of this notebook. I am told this notebook is not designed to run inside **Kubeflow Notebook Servers**.  I have verified this by running this notebook inside the notebook server using default image.

Reply:
I see, thanks @janeman98!

This PR LGTM from a tech docs point of view. I'll loop @abhi-g in, to check the tech side of things.

Reply:
Thanks @sarahmaddox !